sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/query.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+  submission_date_s3 AS submission_date,
+  default_search_engine,
+  SUM(search_count_all) AS searches,
+  COUNT(DISTINCT(client_id)) AS users
```

Comment:
Do we want any additional dimensions here? For example to be able to break this down by country or anything like that?

Reply:
For this dashboard, this is all the dimensions we need